microshift_configuring/microshift_low_latency/microshift-low-latency.adoc

```diff
@@ -0,0 +1,93 @@
+
+//additional resources for real-time kernel
+[role="_additional-resources"]
+.Additional resources
+* link:https://docs.redhat.com/en/documentation/red_hat_enterprise_linux_for_real_time/9[Red Hat Enterprise Linux for Real Time 9] (RHEL documentation)
+* link:https://osbuild.org/docs/on-premises/installation/managing-repositories/#using-repositories-that-require-subscription[Using repositories that require subscription] (osbuild documentation)
+* link:https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html/composing_a_customized_rhel_system_image/creating-system-images-with-composer-command-line-interface_composing-a-customized-rhel-system-image#building-rhel-images-by-using-the-real-time-kernel_creating-system-images-with-composer-command-line-interface[Building RHEL images by using the real-time kernel] for more information.
+* link:https://docs.redhat.com/en/documentation/red_hat_enterprise_linux_for_real_time/9/html/installing_rhel_9_for_real_time/assembly_installing-rhel-for-real-time_installing-rhel-9-for-real-time#post-installation-instructions_assembly_installing-rhel-for-real-time[Post installation instructions] (RHEL for Real Time documentation)
+* xref:../../microshift_install/microshift-embed-in-rpm-ostree.adoc#microshift-embed-in-rpm-ostree[Embedding in a {op-system-ostree} image]
+
+* link:https://access.redhat.com/solutions/4096521[FAQ about RHEL for Real Time (kernel-rt)]
+//Can likely remove KCS after 2024 as it may go out of date
+
+include::modules/microshift-low-latency-prepare-workload.adoc[leveloffset=+1]
+
+//additional resources for preparing the workload
+[role="_additional-resources"]
+.Additional resources
+
+* link:https://docs.openshift.com/container-platform/4.16/scalability_and_performance/low_latency_tuning/cnf-provisioning-low-latency-workloads.html#cnf-configuring-high-priority-workload-pods_cnf-provisioning-low-latency[Disabling power saving mode for high priority pods] (Red Hat OpenShift Container Platform documentation)
+
+* link:https://docs.openshift.com/container-platform/4.16/scalability_and_performance/low_latency_tuning/cnf-provisioning-low-latency-workloads.html#cnf-disabling-cpu-cfs-quota_cnf-provisioning-low-latency[Disabling CPU CFS quota] (Red Hat OpenShift Container Platform documentation)
+
+* link:https://docs.openshift.com/container-platform/4.16/scalability_and_performance/low_latency_tuning/cnf-provisioning-low-latency-workloads.html#cnf-disabling-interrupt-processing-for-individual-pods_cnf-provisioning-low-latency[Disabling interrupt processing for CPUs where pinned containers are running] (Red Hat OpenShift Container Platform documentation)
+
+include::modules/microshift-low-latency-rhel-edge-blueprint-rtk.adoc[leveloffset=+1]
+
+//additional resources for workload partitioning
+//[role="_additional-resources"]
+//.Additional resources
```

Comment:
Why are you hiding the Additional resources heading? It looks odd in the preview.

Reply:
I just overlooked it after I added the link.